plugin/ranger.vim

```diff
@@ -39,7 +39,11 @@ if has('nvim')
       endtry
     endfunction
     enew
-    call termopen('ranger --choosefiles=/tmp/chosenfile --selectfile="' . currentPath . '"', rangerCallback)
+    if isdirectory(currentPath)
```

Comment:
Why not have two distinct functions? One that  opens with the focus on the parameter and one that opens with the current working directory set to the parameter. The first should raise an error if the parameter does not exist in the file system. The second should raise an error if the parameter is not a directory in the file system.

Reply:
In ranger.vim "RangerWorkingDirectory" is the root path of your project and "RangerCurrentDirectory" is the path where you current file is located and "RangerCurrentFile" that open the file's directory and select the file you are editing.

This issue is when you try to open current file, but it opens the parent directory selecting the path you are, whit this fix you can use "RangerCurrentFile" to open the file's directory and select the file if exists, else ranger will open the directory where the file must be.

So, I think that functions exists already with "RangerCurrentFile".

Reply:
I am talking about low level functions not API functions.

Reply:
I don't understand, I think there is no need to make another function just to validate a file or directory existence, because if you use the ranger commands it will check for the selected file or by the default the directory of the file if is not found, maybe a error message can be good.

Is that what you mean?

Reply:
Nevermind, what you wrote works.

Reply:
Thank you!